spec/robots/dor_repo/caption/start_caption_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Robots::DorRepo::Caption::StartCaption do
+  let(:druid) { 'druid:bb222cc3333' }
+  let(:robot) { described_class.new }
+
+  let(:object) { build(:dro, id: druid) }
+  let(:workspace_client) { instance_double(Dor::Services::Client::Workspace, create: nil) }
```

Comment:
I have the same question here: should `create()` be getting called, because it appears that the tests still pass when removing `create: nil`.

Reply:
yes, good call... i removed the mocked method in the new tests where it is not needed